shanoir-ng-studies/src/main/java/org/shanoir/ng/subject/controler/SubjectApiController.java

```diff
@@ -92,11 +93,22 @@ public ResponseEntity<SubjectDTO> findSubjectById(
 	}
 
 	@Override
-	public ResponseEntity<List<SubjectDTO>> findSubjects() {
-		final List<Subject> subjects = subjectService.findAll();
+	public ResponseEntity<List<SubjectDTO>> findSubjects(boolean preclinical, boolean clinical) {
```

Comment:
Is this necessary to have two booleans when a subject is either clinical or preclinical ?

Reply:
Hi @julien-louis,

I thought of 3 possible uses cases :
- we want all subject (default values : clinical = true & preclinical = true)
- we want only clinical (clinical = true & preclinical = false)
- we want only preclinical (clinical = false & preclinical = true)

However, for now, API is only used for getting all subjects, or just preclinical one.

Do you think of a more elegant solution ?

Reply:
Ok I see now, seems fine, thank you